cmd/installer/pkg/install/constants.go

```diff
@@ -0,0 +1,38 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package install
```

Comment:
refactoring in the package:

* pulled out constants
* extracted `Target` as `target.go`
* extracted our default partition scheme `Target`s as method `NewXXXTarget`